src/server/JasmineGraphServer.cpp

```diff
@@ -66,6 +67,8 @@ int JasmineGraphServer::run(std::string profile, std::string masterIp, int numbe
     this->numberOfWorkers = numberofWorkers;
     this->workerHosts = workerIps;
     init();
+    masterPortVector.push_back(Conts::JASMINEGRAPH_FRONTEND_PORT);
+    addInstanceDetailsToPerformanceDB(masterHost,masterPortVector,"true");
```

Comment:
Lets add spaces between the parameters. Please follow the same best practice in all other places.